project/cmd.go

```diff
@@ -23,10 +24,16 @@ func AddCommands(parent *cobra.Command, group *cobra.Group) {
 		cmd.Args = cobra.NoArgs
 	})
 
-	shared.AddCommand(project, Clone, "clone <existingProjectId> <newProjectName> [-t tag]", "Clone an existing project along with all settings and configurations", func(cmd *cobra.Command) {
+	shared.AddCommand(project, Clone, "clone <existingProjectId> <newProjectName> [-e environment] [--tags tag,...]", "Clone an existing project along with all settings and configurations", func(cmd *cobra.Command) {
```

Comment:
Don't we need backward compatibility here?

Reply:
There's backwards compatibility with deprecation in the other file.